src/app/ASKS/ask/index.js

```diff
@@ -377,22 +377,24 @@ const Ask = ({
           config={config}
         />
       )}
-      {component === 'text' && (
-        <Text.Write
-          questionCode={questionCode}
-          mandatory={mandatory}
-          data={data}
-          onSendAnswer={onSendAnswer}
-          regexPattern={regexPattern}
-          errorMessage={errorMessage}
-          attributeCode={attributeCode}
-          targetCode={targetCode}
-          parentCode={parentCode}
-          placeholderName={placeholderName}
-          clientId={clientId}
-          inputmask={inputmask}
-        />
-      )}
+      {component === 'text' ||
```

Comment:
should be: 
               {(component === 'text' || component === 'time') && (
        <Text.Write
          ...
        />
      )}

not what you have currently as :
{component === 'text' ||
        (component === 'time' && (
          <Text.Write
            ...
          />
        ))}

Reply:
fixed in latest commit